types/screener.d.ts

```diff
@@ -0,0 +1,66 @@
+//
+// Typings for `screener-runner`
+//
+
+type ScreenerRunnerKeys = {
+  alt: string
+  control: string
+  enter: string
+  escape: string
+  return: string
+  shift: string
+  tab: string
+  leftArrow: string
+  upArrow: string
+  rightArrow: string
+  downArrow: string
+}
+
+type ScreenerRunnerSteps = {
```

Comment:
rename to `ScreenerStepComposer` or `ScreenerStepBuilder`? Otherwise it is very hard to read the last two declarations of this file:
- ScreenerSteps -> many of ScreenerStep
- ScreenerStep -> (steps, keys) // ?

Reply:
`ScreenerStepBuilder` looks like the correct one 👍